inc/saitypes.h

```diff
@@ -1428,6 +1428,21 @@ typedef enum _sai_stats_mode_t
      * @brief Read and clear after reading
      */
     SAI_STATS_MODE_READ_AND_CLEAR = 1 << 1,
+
+    /**
+     * @brief Build read statistics
+     */
+    SAI_STATS_MODE_BULK_READ = 1 << 2,
```

Comment:
What is the expected behavior of sai_bulk_object_get_stats/sai_bulk_object_clear_stats \_In\_ sai_stats_mode_t mode is (or isn't) with respect to these new enums? Must the new BULK_MODE enums be passed?

Reply:
sai_stat_capability_t should be queried for supported stats mode.

Based on that sai_stats_mode_t flag is passed in the BULK API. Since mode is at stats_id granularity, caller must ensure that all the stats_id are of same stats_mode type.

Reply:
ack, thank you